CONVENTIONS.md

```diff
@@ -0,0 +1,45 @@
+# Convetions
+
+This document gives coding conventions for the Superface capabilities created in [Station repository](https://github.com/superfaceai/station).
+
+## Profile
+
+### Structure
+
+- Use 2 spaces per indentation level.
+- Profile should contain one usecase.
+- Usecase must have error defined.
+- Usecase must have defined safety.
+- Fields must use [required](https://superface.ai/docs/comlink/profile#RequiredField) declaration.
+- Named fields and models shouldn't be used, unless they are reused. _(Better Profile rendering)_
+
+### Naming conventions
+
+- Profile filename must use `-` (dash) as delimiter.
+- Usecase name must use **PascalCase** format.
+- Field name must use **camelCase** format.
+
+### Descriptions
+
+- Use block string literal `"""` when writing title and description.
+- Profile must have title and description.
+- Every usecase must have title and description.
+- Every field must have title and description.
+
+### Map
+
+We need your feedback.
+
+### Provider
+
+- Provider name must use `-` (dash) as delimiter.
+- Provider Id should be short and descriptive.
+- Default service Id for versioned APIs should match the version.
+- Security Scheme Id must use `-` (dash) as delimiter.
+- Security Scheme Id should be named same as in provider documentation.
+
+### Tests
+
+- Capability must have automated tests.
+- Test must use network recording.
+- Test must use snapshot for asserting.
```

Comment:
Maybe add "TODO" and link to @martinalbert testing set when available

Reply:
Finally, maybe add a link to a role model profile?

Reply:
I planned to add how to write tests in the README and update existing. Until it is ready, I will replace last two point with todo to add link to readme how tests should be done.

Ok I will link it